### DISCUSSION TITLE
Not able to utilise the plugin

### DISCUSSION THREAD

**Discussion by DiljeetKaur-ELS** [ImFlog/schema-registry-plugin#56] [Q&A] [answered]
Hello @ImFlog @bspeakmon @akhil-karanth @codyoss ,
I was trying to utilize the plugin for downloading the schema from the Schema Registry but unfortunately I'm not able to setup the plugin.
The snippet of my `build.gradle.kts.`:

> plugins {
>     id("java")
>     id("org.springframework.boot") version "2.4.4"
>     id("io.spring.dependency-management") version "1.0.11.RELEASE"
>     id("com.github.imflog.kafka-schema-registry-gradle-plugin") version "1.2.0"
>     id("com.github.davidmc24.gradle.plugin.avro") version "1.0.0"
> }
> 
> repositories {
>     mavenCentral()
>     jcenter()
>     maven {
>         url = uri("https://plugins.gradle.org/m2/")
>     }
>     maven {
>         name = "confluent"
>         url = uri("https://packages.confluent.io/maven/")
>     }
> }

When I do `gradle clean build`, I get error:
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve com.github.imflog:kafka-schema-registry-gradle-plugin:1.2.0.
     Required by:
         project : > com.github.imflog.kafka-schema-registry-gradle-plugin:com.github.imflog.kafka-schema-registry-gradle-plugin.gradle.plugin:1.2.0
      > Could not resolve com.github.imflog:kafka-schema-registry-gradle-plugin:1.2.0.
         > Could not parse POM https://plugins.gradle.org/m2/com/github/imflog/kafka-schema-registry-gradle-plugin/1.2.0/kafka-schema-registry-gradle-plugin-1.2.0.pom
            > Could not find io.confluent:kafka-schema-registry-parent:6.1.0.
              Searched in the following locations:
                - https://plugins.gradle.org/m2/io/confluent/kafka-schema-registry-parent/6.1.0/kafka-schema-registry-parent-6.1.0.pom
              If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.

**Comment by ImFlog** [comment, marked as answer]
Hello @DiljeetKaur-ELS,
Thank you for the interest on the plugin.
The reason it is not working is because you are defining repositories for your build and not repositories for the plugins.
As stated in the readme you need to add the repository in a `buildscript {}` block. I tried on my computer and here is what your build.gradle.kts should look like:
```kotlin
plugins {
	id("java")
	id("org.springframework.boot") version "2.4.4"
	id("io.spring.dependency-management") version "1.0.11.RELEASE"
	id("com.github.imflog.kafka-schema-registry-gradle-plugin") version "1.2.0"
	id("com.github.davidmc24.gradle.plugin.avro") version "1.0.0"
}

// The buildscript block was the missing part.
buildscript {
	repositories {
		gradlePluginPortal()
		maven {
			name = "confluent"
			url = uri("https://packages.confluent.io/maven/")
		}
		maven {
			url = uri("https://jitpack.io")
		}
	}
}


repositories {
	mavenCentral()
	jcenter()
        // Add the repository according to the libraries you want to use in your code.
}
```

Hope it helps.
